### PR TITLE
Rename the 'not' operator to 'i32.eqz' and add 'i64.eqz'.

### PR DESCRIPTION
**Pull request by JSStats**
It fitted the 'convert' type of operator so this was used - not sure if it's what you would want.

### CODE REVIEW

**Comment by binji** [comment]
Looks like a good start, but the build is failing.

The convert type is probably the best bet, yeah -- it looks like the spec repo added a new `TEST` type, but I'd rather not add a new expression type for just this.

Also, I have a CL locally that removes all the operator type stuff and just uses the opcode, FYI

**Comment by JSStats** [comment]
Rebased. The failure looks like the d8 version might be out of sync, and it works locally, or do some 64 bit operators not yet work on d8-32bit and is this used in checks here?

**Comment by binji** [comment]
Oh yeah, the version of d8 is pinned, so it probably doesn't have `i64.eqz`. I'll update it. Also, it looks like you'll need to update the lexer to get `i64.eqz` as well, that's failing on the gcc bot.

**Comment by binji** [comment]
OK, updated v8.

**Comment by JSStats** [comment]
Thank you. Updated. ptal.

File: src/wasm-common.h [hunk 1]
```diff
@@ -142,7 +142,7 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
   V(I32, ___, 0, 0x57, I32_CLZ)             \
   V(I32, ___, 0, 0x58, I32_CTZ)             \
   V(I32, ___, 0, 0x59, I32_POPCNT)          \
-  V(I32, ___, 0, 0x5a, I32_EQZ)             \
+  V(I32, ___, 0, 0x5a, I32_EQZ_I32)         \
```
Comment (binji) [inline src/wasm-common.h]: I'd rather keep the names consistent with the spec names, so `I32_EQZ` and `I64_EQZ`

Comment (binji) [inline src/wasm-common.h]: I'm surprised this works, the convert should use the second column as the type to convert from.

**Comment by JSStats** [comment]
Thank you. Updated. ptal.

**Comment by binji** [comment]
lgtm, thanks

**Pull request merged**